libs/components/src/lib/rich-data-table/stories/vl-rich-data-table.stories.ts

```diff
@@ -1,76 +1,68 @@
     const data =
-        '{"data": [{ "id" : 0, "name" : "Project #1" , "owner" : "Jan Jansens" }, { "id" : 1, "name" : "Project #2" , "owner" : "Jan Jansens" }]}';
+        '{"data": [{ "id" : 0, "name" : "Project #1" , "owner" : "Jan Jansens" }, { "id" : 1, "name" : "Project #2" , "owner" : "Marie Vermeersch" }]}';
 
-    const tableSorter = (table: any) => {
-        const originalTableData = [...table.data.data];
-        return (event: any) => {
-            const { sorting } = event.detail;
-            const table = event.target;
-            if (sorting) {
-                table.data = {
-                    data: [...originalTableData].sort((firstEl, secondEl) => {
-                        for (let i = 0; i < sorting.length; i++) {
-                            const criteria = sorting[i];
-                            const firstValue = firstEl[criteria.name];
-                            const secondValue = secondEl[criteria.name];
-                            const isAscending = criteria.direction === 'asc';
-                            if (firstValue < secondValue) {
-                                return isAscending ? -1 : 1;
-                            } else if (firstValue > secondValue) {
-                                return isAscending ? 1 : -1;
-                            }
-                        }
-                        return 0;
-                    }),
-                    sorting,
-                };
-            } else {
-                table.data = originalTableData;
-            }
-        };
-    };
-    customElements.whenDefined('vl-rich-data-table').then(() => {
-        const table: any = document.querySelector('#rich-data-table-sorting');
-        table.addEventListener('change', tableSorter(table));
-    });
+    return html`
+        <vl-rich-data-table
+            data-vl-data="${data}"
+            ?data-vl-collapsed-m=${collapsedM}
+            ?data-vl-collapsed-s=${collapsedS}
+            ?data-vl-collapsed-xs=${collapsedXS}
+        >
+            <vl-rich-data-field data-vl-name="id" data-vl-label="ID" data-vl-selector="id"></vl-rich-data-field>
+            <vl-rich-data-field data-vl-name="name" data-vl-label="Naam" data-vl-selector="name"></vl-rich-data-field>
+            <vl-rich-data-field data-vl-name="owner" data-vl-selector="owner">
+                <template slot="label">
+                    <span>Eigenaar</span>
+                </template>
+            </vl-rich-data-field>
+        </vl-rich-data-table>
+    `;
+};
+export const RichDataTableDefault = TemplateBase.bind({});
+RichDataTableDefault.storyName = 'vl-rich-data-table - default';
+RichDataTableDefault.args = richDataTableArgs;
+RichDataTableDefault.args = {
+    // @ts-ignore: Negeer de type-check van toggleText.
+    filterCloseable: nothing,
+    // @ts-ignore: Negeer de type-check van toggleText.
+    filterClosed: nothing,
+};
```

Comment:
Kan je in plaats van argumenten die je niet wilt hebben te schrappen de argumenten die je wel wilt hebben toevoegen?
In de default export vanboven in de file moet je dan de args weglaten.
Voor een story die alle args gebruikt moet je dan het volgende doen:
`<storyName>.args = richDataTableArgs`

Reply:
done